14-RemoteRepository/README.md

```diff
@@ -0,0 +1,145 @@
+# Working with remote repository
+
+## Introduction
+
+Once we have our remote repository configured we would be able to work with it.
+
+The list of commands below allows to work with remote repository.
+
+### Upstream
```

Comment:
This section is too abstract because you have not explained how push/fetch works. You should:
- change the explanation of git push not to use set upstream (-u)
- explain what upstream is (i.e. move this section after git push w/o upstream)
- give an example how upstream can make the command easier
I think this deserves a bit more detailed description. The user should know that local and remote branches are independent and upstream is just a way how to make things easier (i.e it is not required to have the same branch names/hierarchy)